README.md

```diff
@@ -14,25 +15,24 @@
 </p>
 
 <p align="center">
-   The Design System that powers Razorpay
+   <b>Design System that powers Razorpay⚡  </b>
 <p align="center">
 
 ### [`@razorpay/blade`](https://github.com/razorpay/blade/tree/master/packages/blade)
+![Github Repo Size](https://img.shields.io/github/repo-size/razorpay/blade?style=for-the-badge&color=aqua)
+
+
 
 This package is under active development.
 
-- [Documentation](https://master--61c19ee8d3d282003ac1d81c.chromatic.com)
-- [Installation Guide](https://master--61c19ee8d3d282003ac1d81c.chromatic.com/?path=/docs/guides-installation--page)
-- [Contributing guidelines](https://github.com/razorpay/blade/blob/master/CONTRIBUTING.md)
+[Documentation](https://master--61c19ee8d3d282003ac1d81c.chromatic.com) &nbsp;&nbsp;&nbsp; [Installation Guide](https://master--61c19ee8d3d282003ac1d81c.chromatic.com/?path=/docs/guides-installation--page)&nbsp;&nbsp;&nbsp; [Contributing guidelines](https://github.com/razorpay/blade/blob/master/CONTRIBUTING.md)
+
 
----
 
-> **Note**
->
-> The blade-old package has been moved to it's own repo.
-> [github.com/razorpay/blade-old](https://github.com/razorpay/blade-old) (Only accessible to Razorpay employees)
->
-> It is under **maintenance** and it won't have any major releases.
+## Note
```

Comment:
this was on purpose because GitHub markdown highlights this content as below and we want this as information rather than a heading in its own. Can you revert this?
<img width="863" alt="Screenshot 2023-01-16 at 10 45 25 AM" src="https://user-images.githubusercontent.com/11384858/212603605-9c903e68-b872-443c-9fb3-3df194ec6965.png">